libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -232,6 +234,25 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     AP_GROUPINFO("_FIELD_ELV", 22, AP_Baro, _field_elevation, 0),
 #endif
 #endif
+
+#if APM_BUILD_COPTER_OR_HELI || APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    // @Param: _ALTERR_MAX
+    // @DisplayName: Maximum altitude error
+    // @Description: This is the maximum acceptable altitude discrepancy between GPS altitude and barometric presssure altitude calculated against a standard atmosphere for arming checks to pass. If you are getting an arming error due to this parameter then you may have a faulty or substituted barometer. A common issue is vendors replacing a MS5611 in a "Pixhawk" with a MS5607. If you have that issue then please see BARO_OPTIONS parameter to force the MS5611 to be treated as a MS5607
+    // @Units: m
+    // @Increment: 1
+    // @Range: 0 5000
+    // @User: Advanced
+    AP_GROUPINFO("_ALTERR_MAX", 23, AP_Baro, _max_gps_alt_error, 2000),
```

Comment:
for the variable name, I'd normally make it match the parameter name.  So _alt_error_max or something similar... at least "max" should be at the end if possible.

Reply:
done